app/src/main/java/com/sunasterisk/fooddaily/ui/adapter/FoodViewHolder.kt

```diff
@@ -1,27 +1,69 @@
 package com.sunasterisk.fooddaily.ui.adapter
 
+import android.annotation.SuppressLint
 import android.view.View
 import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.resource.bitmap.CenterCrop
+import com.bumptech.glide.load.resource.bitmap.RoundedCorners
+import com.bumptech.glide.request.RequestOptions
+import com.sunasterisk.fooddaily.R
 import com.sunasterisk.fooddaily.data.model.FoodDetail
+import kotlinx.android.synthetic.main.item_daily_menu_food.view.*
+import kotlinx.android.synthetic.main.item_other_food_grid.view.*
+import kotlinx.android.synthetic.main.item_other_food_list.view.*
 
-abstract class FoodViewHolder(view: View): RecyclerView.ViewHolder(view) {
+abstract class FoodViewHolder(view: View) : RecyclerView.ViewHolder(view) {
     abstract fun bind(food: FoodDetail)
 }
 
 class DailyMenuViewHolder(view: View) : FoodViewHolder(view) {
+    @SuppressLint("SetTextI18n")
     override fun bind(food: FoodDetail) {
-        //TODO bind data from foodList to DailyMenu
+        itemView.textFoodNameDailyMenu.text = food.title
+        itemView.textDailyMenuPrice.text = food.price
+        itemView.textDailyMenuReadyMinutes.text =
+            food.readyMinutes + itemView.resources.getString(R.string.value_time_required)
```

Comment:
Tham khảo 
https://stackoverflow.com/questions/12627457/format-statement-in-a-string-resource-file
Để viết lại đoạn này nhé

Reply:
đoạn này mình cũng viết vậy mà nhỉ